torch/quantization/observer.py

```diff
@@ -64,13 +70,21 @@ def calculate_qparams(self, **kwargs):
 
 
 class _ObserverBase(Observer):
-    r"""
-    Common base for all qint/quint8 observers
+    r"""Internal common base for all qint/quint8 observers.
+
+    This base is for commonly used paramters used internally.
+    Users should use `~torch.quantization.observer.Observer` as a base class for
+    custom observers.
+
+    Args:
+        dtype: Quantized data type
```

Comment:
Can you also specify valid values for dtype and qscheme? For example., qint32 is not valid.